packages/expo-doctor/src/checks/AppConfigFieldsNotSyncedToNativeProjectsCheck.ts

```diff
@@ -42,6 +42,7 @@ export class AppConfigFieldsNotSyncedToNativeProjectsCheck implements DoctorChec
       ? 'EAS Build will not sync the following properties:'
```

Comment:
The line right above here kind of sets us down an EAS-centric or non-EAS centric path. Even though it's unlikely that you have no eas.json but you do have an easignore, it's possible, and it's technically more correct if we adjust our ignore-checking to use or not use easignore based on whether or not eas.json exists. Same thing may also apply to `StoreCompatibilityCheck`.

Reply:
Got it, so our function:
```
export async function isFileIgnoredAsync(
  filePath: string,
  checkEasignore: boolean = true
): Promise<boolean> {
```
Should be updated to something like:
```
export async function isFileIgnoredAsync(
  filePath: string,
  checkEasignore: boolean = fs.existsSync('eas.json');
): Promise<boolean> {
```

We can still overwrite it by setting the flag true or false when calling the function